content/en/community/hackathons/sessions/baby-steps/work/01-echo-back/main.c

```diff
@@ -88,7 +101,9 @@ int main(int argc, char* argv[])
 		bytes_received = receive_and_send(conn_fd, conn_fd);
```

Comment:
Add a TODO item here to add the right parameter to the `receive_and_send` function.